specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/LinkedService.json

```diff
@@ -3418,6 +3418,18 @@
         "scope": {
           "type": "object",
           "description": "The scope of the access required. It describes what kind of access will be requested. Type: string (or Expression with resultType string)."
+        },
+        "servicePrincipalCredentialType": {
+          "type": "object",
```

Comment:
Should it be a string type?

Reply:
ADF support expression, so it could be a object type.
https://learn.microsoft.com/en-us/azure/data-factory/control-flow-expression-language-functions